### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by ImWichop** [codewars/codewars.com#2526] [Q&A]
Hi Mods, Could you explain me why My account is restricted? I usually do the challenges in my own editor and just copy and paste the code on Codewars. I hope you can help me.

**Comment by ImWichop** [comment]
My nickname is luiscunosquare

**Comment by Blind4Basics** [comment]
Hi,

You posted solutions you didn't author, hence the restriction, yes.

**Reply by ImWichop** [reply]
And could you remove that solution and restore my account please?

**Reply by Blind4Basics** [reply]
better to delete your account and start fresh. And this time, don't use solutions you find on the web.

**Reply by ImWichop** [reply]
Alright!
